tests/vmi_lifecycle_test.go

```diff
@@ -129,21 +129,6 @@ var _ = Describe("VMIlifecycle", func() {
 				500*time.Millisecond).
 				Should(ContainSubstring("info : hostname: " + dns.SanitizeHostname(vmi)))
 		})
-
-		It("should reject POST if schema is invalid", func() {
```

Comment:
why was this removed?

Reply:
this one interesting, so the scenario is:
- defines new VMI
- changes `domain` under spec to `not-a-domain`
- creates VMI, but first, it sends VMI to mutating webhook that set defaults, so it will add `domain` section to VMI, and now under VMI spec we have two sections. `domain` and `not-domain`, so API server the is valid(it just ignore `not-domain` section)
I tried to figure out how can save this test, but I lack of ideas.

Reply:
Ha, that's funny. I'm fine with just removing the test then.